x-pack/plugins/index_management/public/lib/render_badges.js

```diff
@@ -18,19 +18,21 @@ export const renderBadges = (index, filterChanged) => {
           && filterExpression
           && filterChanged(EuiSearchBar.Query.parse(filterExpression));
       };
+      const ariaLabel = i18n.translate(
+        'xpack.idxMgmt.badgeAriaLabel',
+        {
+          defaultMessage: '{label}. Select to filter on this.',
+          values: { label },
+        },
+      );
       badgeLabels.push(
         <Fragment key={label}>
           {' '}
           <EuiBadge
             color={color}
             onClick={clickHandler}
-            aria-label={i18n.translate(
-              'xpack.idxMgmt.badgeAriaLabel',
-              {
-                defaultMessage: '{label}. Select to filter on this.',
-                values: { label },
-              },
-            )}
+            onClickAriaLabel={ariaLabel}
+            aria-label={ariaLabel}
```

Comment:
Looks like `onClickAriaLabel` gets applied to the badge internally, so I think we can drop `aria-label` now.

Reply:
removed the aria-label